packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -622,22 +622,25 @@ class Actions extends StatefulWidget {
   ///
   /// The `context`, `intent` and `nullOk` arguments must not be null.
   ///
-  /// If the given `intent` isn't found in the first [Actions.actions] map, then
-  /// it will look to the next [Actions] widget in the hierarchy until it
-  /// reaches the root.
-  ///
-  /// Will throw if no ambient [Actions] widget is found, or if the given
-  /// `intent` doesn't map to an action in any of the [Actions.actions] maps
-  /// that are found.
-  ///
-  /// Setting `nullOk` to true means that if no ambient [Actions] widget is
-  /// found, then this method will return null instead of throwing.
-  ///
-  /// Returns the result of invoking the action's [Action.invoke] method. If
-  /// no action mapping was found for the specified intent, or if the action
-  /// that was found was disabled, then this returns null. Callers can detect
-  /// whether or not the action is available (found, and not disabled) using
-  /// [Actions.find] with its `nullOk` argument set to true.
+  /// If the given `intent` doesn't map to an action, or doesn't map to one that
+  /// returns true for [Action.isEnabled] in an [Actions.actions] map it finds,
+  /// then it will look to the next ancestor [Actions] widget in the hierarchy
+  /// until it reaches the root.
+  ///
+  /// In debug mode, If `nullOk` is false, this method will throw an exception
```

Comment:
Do we still plan to remove this? (not in this PR, of course)

Reply:
Yeah, I thought about that when I was doing my pass a few weeks ago. In this case, I think it's somewhat useful functionality (it gives good feedback for a runtime error), and removing it still won't mean that we can convert the return type to a non-nullable value, since we want to be able to return null from the invocation of the action.

Ideally, I'd name it something else like "actionNotFoundOk" or something, since "nullOk" is really kind of a misnomer: it's always OK to return null here.

Reply:
I might also change the default to be "true", since that seems more likely to be the desired state, really.